nityam_kumar/PostgresAssignment/src/db/validateSchema/studentSchema.ts

```diff
@@ -0,0 +1,11 @@
+import Joi from "joi";
+
+const studentSchema = Joi.object().keys({
+  fname: Joi.string().trim().required().min(3).max(25),
+  lname: Joi.string().trim().required().min(3).max(25),
+  sex: Joi.string().lowercase().trim().valid(null, "O", "M", "F"),
+  age: Joi.number().required().min(1).max(110),
```

Comment:
note : age can be derived using dob

Reply:
added dob attribute for student and teacher